File: frontend/src/fixtures/reverse/reverse.js
#!/usr/bin/env node

const args = process.argv.slice(2);
const str = args.join(" ");

/**
 * Reverse a string
 * @param {string} str - The string to be reversed
 * @returns {string} - The reversed string
 */
const reverse = (str) => str.split("").reverse().join("");
console.log(reverse(str));

File: funk/reverse.go
package funk

func Reverse[T comparable](slice []T) []T {
	s := append([]T{}, slice...)

	for i, j := 0, len(s)-1; i < j; i, j = i+1, j-1 {
		s[i], s[j] = s[j], s[i]
	}

	return s
}
